explorer/interpreter/action.h

```diff
@@ -42,8 +42,7 @@ class RuntimeScope {
   RuntimeScope(RuntimeScope&&) noexcept;
   auto operator=(RuntimeScope&&) noexcept -> RuntimeScope&;
 
-  // Deallocates any allocations in this scope from `heap`.
-  ~RuntimeScope();
+  ~RuntimeScope() = default;
```

Comment:
```suggestion
```

Can this be deleted? It looks like maybe it doesn't need to be explicit?

Reply:
Can do yes.

I preferred to keep it given we should be defining the big 3, clarifying that we want the default implementation on purpose. i.e. we didn't forget to define it.

Reply:
FWIW my leaning would still be to delete, I don't have the "forgot to define a destructor" instinct. But I did approve, so do as you like.